### PR TITLE
Apply Prettier to SCSS files

### LINKED ISSUE #5431: Apply Prettier to SCSS files
Prettier supports SCSS/CSS, so we could use to prettier-ize our style files.

Our `stylelint` config will conflict with Prettier because we use double quotes in our SCSS files, but not in our JS files. This seems inconsistent, so I guess, we'll need to update the rule in `.stylelintrc` to use `"string-quotes": "single"`.

### PR DESCRIPTION
**Pull request by willdurand**
Fix #5431

### CODE REVIEW

File: src/amo/css/inc/vars.scss [hunk 1]
```diff
@@ -14,12 +14,34 @@ $accent-color: #ebebeb;
...
+      min-width: 720px,
+    ),
+  extraLarge: only screen and
+    (
+      min-width: 900px,
+    ),
+  extraExtraLarge: only screen and
+    (
+      min-width: 1150px,
+    )
```
Comment (willdurand) [inline src/amo/css/inc/vars.scss]: This is weird.. 🤷‍♂️

File: src/ui/components/LoadingText/styles.scss [hunk 2]
```diff
@@ -35,6 +35,10 @@
 
 @for $delay from 1 through 3 {
   .LoadingText--delay-#{$delay} {
-    animation: placeHolderShimmer #{$delay * 1.5}s infinite cubic-bezier(0.65, 0.05, 0.36, 1);
+    animation: placeHolderShimmer
+      #{$delay *
+      1.5}s
```
Comment (willdurand) [inline src/ui/components/LoadingText/styles.scss]: This is weird too.

**Comment by willdurand** [comment]
I don't know how you feel about that, there are annoying things but it does a decent job otherwise..

File: src/amo/css/inc/vars.scss [hunk 3]
```diff
@@ -14,12 +14,34 @@ $accent-color: #ebebeb;
 // it with `@include respond-to(medium) { [...] }`.
 $breakpoints: (
   // This breakpoint exists for tweaking of very small screens.
-  extraSmall: only screen and (max-width: 349px),
-  mediumOnly: only screen and (min-width: 500px) and (max-width: 719px),
-  medium: only screen and (min-width: 500px),
-  large: only screen and (min-width: 720px),
-  extraLarge: only screen and (min-width: 900px),
-  extraExtraLarge: only screen and (min-width: 1150px),
+    extraSmall: only screen and
```
Comment (kumar303) [inline src/amo/css/inc/vars.scss]: Huh. This seems like it should not be indented.

**Pull request merged**